pkl-formatter/src/test/files/FormatterSnippetTests/output/for-amend.pkl

```diff
@@ -0,0 +1,14 @@
+module forgenerator.amend
+
+local app = new {
+  some = "a"
+  field = "b"
+}
+
+l = new Listing {
+  for (_item in (app) {
```

Comment:
Honestly, I'm not sure how this _should_ look like. But what the current formatter produces is not ideal:

```pkl
        for
        (
        _ingress: Ingress
        in
        (app) {
```